@navikt/core/css/darkside/button.darkside.css

```diff
@@ -2,5 +2,498 @@
+  .navds-button.navds-button--primary {
+    background-color: highlight;
+    color: highlighttext;
+  }
+
+  .navds-button.navds-button--primary span {
+    forced-color-adjust: none;
+  }
+}
+
+.navds-button--primary:hover {
+  background-color: var(--a-bg-accent-strong-hover);
+}
+
+.navds-button--primary:active {
+  background-color: var(--a-bg-accent-strong-pressed);
+}
+
+.navds-button--primary:focus-visible {
+  outline: 2px solid transparent;
+  outline-offset: 2px;
+  box-shadow:
+    inset 0 0 0 1px var(--a-surface-default),
+    var(--a-shadow-focus);
+}
+@media (forced-colors: active) {
+  .navds-button--primary:focus-visible {
+    box-shadow: none;
+  }
+}
+
+@supports not selector(:focus-visible) {
+  .navds-button--primary:focus {
+    box-shadow:
+      inset 0 0 0 1px var(--a-surface-default),
+      var(--a-shadow-focus);
+  }
+}
+
+.navds-button--primary:hover:where(:disabled, .navds-button--disabled),
+.navds-button--primary:active:where(:disabled, .navds-button--disabled) {
+  background-color: var(--a-bg-accent-strong);
+}
+
+/*************************
+ * .navds-button--primary-neutral *
+ *************************/
+
+.navds-button--primary-neutral {
+  background-color: var(--a-bg-neutral-strong);
+  color: var(--a-text-neutral-contrast);
+}
+
+.navds-button--primary-neutral:hover {
+  background-color: var(--a-bg-neutral-strong-hover);
+}
+
+.navds-button--primary-neutral:active {
+  background-color: var(--a-bg-neutral-strong-pressed);
+}
+
+.navds-button--primary-neutral:focus-visible {
+  box-shadow:
+    inset 0 0 0 1px var(--a-surface-default),
+    var(--a-shadow-focus);
+}
+
+@supports not selector(:focus-visible) {
+  .navds-button--primary-neutral:focus {
+    box-shadow:
+      inset 0 0 0 1px var(--a-surface-default),
+      var(--a-shadow-focus);
+  }
+}
+
+.navds-button--primary-neutral:hover:where(:disabled, .navds-button--disabled),
+.navds-button--primary-neutral:active:where(:disabled, .navds-button--disabled) {
+  background-color: var(--a-bg-neutral-strong);
+}
+
+/**************************
+* .navds-button--secondary *
+ **************************/
+
+.navds-button--secondary {
+  background-color: transparent;
+  color: var(--a-text-accent);
+  box-shadow: inset 0 0 0 2px var(--a-border-accent);
+}
+
+.navds-button--secondary:hover {
+  background-color: var(--a-bg-accent-moderate-hover);
+  color: var(--a-bg-accent-strong-hover);
+  box-shadow: inset 0 0 0 2px var(--a-bg-accent-strong-hover);
+}
+
+.navds-button--secondary:focus-visible {
+  box-shadow:
+    inset 0 0 0 2px var(--a-bg-accent-strong),
+    var(--a-shadow-focus);
+}
+
+@supports not selector(:focus-visible) {
+  .navds-button--secondary:focus {
+    box-shadow:
+      inset 0 0 0 2px var(--a-bg-accent-strong),
+      var(--a-shadow-focus);
+  }
+}
+
+.navds-button--secondary:active {
+  background-color: var(--a-bg-accent-strong-pressed);
+  color: var(--a-text-accent-contrast);
+  box-shadow: none;
+}
+
+/* --------- */
```

Comment:
This marker denotes the current progress of conversion (going from top to bottom)